recipes/sepp/meta.yaml

```diff
@@ -15,24 +15,24 @@ source:
       - nodeps.setup.py.patch
 
 build:
-  number: 4
+  number: 0
   run_exports:
     - {{ pin_subpackage('sepp', max_pin="x") }}
 
 requirements:
   host:
-    - python <=3.9
+    - python <=3.10
     - setuptools
     - pip
     - dendropy <=4.5.1
```

Comment:
Is this correct, since setup.py specifies `dendropy >= 4.6.0` (before patch)?

Reply:
This pin was necessary at the time of https://github.com/smirarab/sepp/pull/126. We might want to check if this is still the case, but maybe do this in a separate PR?

Reply:
Lots going on there, but it reads to me like 4.6.0 was reolved. A separate PR seems reasonable as long as `dendropy <=4.5.1` still works with the latest release.